lib/matplotlib/backend_bases.py

```diff
@@ -270,8 +270,7 @@ def draw_quad_mesh(self, gc, master_transform, meshWidth, meshHeight,
         """
 
         from matplotlib.collections import QuadMesh
-        paths = QuadMesh.convert_mesh_to_paths(
-            meshWidth, meshHeight, coordinates)
+        paths = QuadMesh(coordinates).get_paths()
```

Comment:
Maybe I'm overly sensitive here, but creating a QuadMesh instance is really expensive compared to just converting the path. Thus, I'd keep the static conversion method.

You can use a different name if the deprecation dance is too complicated otherwise.

Reply:
fair point; it's just a matter of using directly `QuadMesh._convert_mesh_to_paths` anyways.